userguide/analytics/userguide_analytics.adoc

```diff
@@ -1,63 +1,595 @@
+* Kill Bill repair items (`REPAIR_ADJ`) are not stored directly in any of these tables, but are combined with the new reparee item as invoice item adjustments (in *analytics_invoice_item_adjustments*). For example, if a recurring item of $20 was repaired into a $5 recurring item, *analytics_invoice_items* would contain the original $20 line item and *analytics_invoice_item_adjustments* would contain a $-15 adjustment item
+* Any invoice item has a revenue recognizable attribute which is used to tell if the item is recognizable from a finance perspective (tied to cash). This is used for credits only today and we assume that any administrator generated credit (`CREDIT_ADJ` and `CBA_ADJ` on their own invoice) is not recognizable. All other credit items, including system generated ones, are.
 
-include::{sourcedir}/analytics/includes/analytics-overview.adoc[]
+*Definitions:*
 
+* *invoice_balance*: invoice_amount_charged + invoice_amount_credited + invoice_amount_adjusted_for_account_credit - (invoice_amount_paid + invoice_amount_refunded). Note that *invoice_amount_adjusted_for_account_credit* is an internal adjustment that applies for account level credits only (CREDIT_ADJ items on their own invoices) and is not represented in Analytics. See https://docs.killbill.io/latest/invoice_examples.html#_invoice_balance[invoice balance] document for further information.
+* *invoice_amount_charged*: sum of all *analytics_invoice_items*, *analytics_invoice_adjustments* and *analytics_invoice_item_adjustments* items for that invoice
+* *invoice_amount_credited*: sum of all *analytics_invoice_credits* items for that invoice (positive if granted, negative if used)
+* *invoice_amount_paid*: sum of all *analytics_payments* items for that invoice
+* *invoice_amount_refunded*: sum of all *analytics_chargebacks* and *analytics_refunds* items for that invoice
+* *invoice_original_amount_charged*: sum of all *analytics_invoice_items* items for that invoice at the time of invoice creation
 
-[[components-analytics-currency]]
-== Currency conversion
+*References:*
 
-include::{sourcedir}/analytics/includes/analytics-currency.adoc[]
+* https://docs.killbill.io/latest/userguide_subscription.html#components-invoice-items[Invoice Items]
 
+* https://docs.killbill.io/latest/invoice_examples.html[Charged Amount and Invoice Balance Calculation examples]
 
-[[components-analytics-accounts]]
-== Accounts
+* https://docs.killbill.io/latest/Kill-Bill-Glossary.html#_i[Kill Bill Glossary]
 
-include::{sourcedir}/analytics/includes/analytics-accounts.adoc[]
+=== Payments
 
+The tables *analytics_payment_auths*, *analytics_payment_captures*, *analytics_payment_purchases*, *analytics_payment_refunds*, *analytics_payment_credits*, *analytics_payment_chargebacks* and *analytics_payment_voids* maintain information about Kill Bill payments, refunds and chargebacks.
```

Comment:
```suggestion
The tables *analytics_payment_auths*, *analytics_payment_captures*, *analytics_payment_purchases*, *analytics_payment_refunds*, *analytics_payment_credits*, *analytics_payment_chargebacks*, and *analytics_payment_voids* maintain information about Kill Bill payments, refunds and chargebacks.
```

Reply:
I would rewrite this to be "The following tables..." and then list them as bullet items.

Reply:
Fixed